src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -836,7 +836,7 @@ public void AnalyzeCommand_EncodedPaths()
         {
             var logger = new MemoryStreamSarifLogger(dataToInsert: OptionallyEmittedData.Hashes);
             var command = new TestMultithreadedAnalyzeCommand();
-            var uri = new Uri("/new folder/0%1%20%.txt", UriKind.Relative);
+            var uri = new Uri("/new%20folder/0%1%20%.txt", UriKind.Relative);
```

Comment:
Recommend breaking this out into a loop that tests the directory case and the file case separately rather than together.

Reply:
Add more scan targets.